OpenRA.Game/Traits/RejectsOrders.cs

```diff
@@ -41,5 +41,17 @@ public static bool AcceptsOrder(this Actor self, string orderString)
 			var r = self.TraitOrDefault<RejectsOrders>();
 			return r == null || r.Except.Contains(orderString);
 		}
+
+		public static void AddOrder(this Actor self, string orderString)
+		{
+			var r = self.TraitOrDefault<RejectsOrders>();
+			r.Except.Add(orderString);
```

Comment:
These will throw NullReferenceExceptions if the actor doesn't have RejectsOrders.